xbmc/commons/Exception.cpp

```diff
@@ -19,11 +19,15 @@
  */
 
 #include "Exception.h"
+#include "utils/log.h"
 
 namespace XbmcCommons
 {
-  ILogger* Exception::logger = NULL;
-
   Exception::~Exception() = default;
+
+  void Exception::LogThrowMessage(const char* prefix) const
+  {
+  	CLog::Log(LOGERROR,"EXCEPTION Thrown (%s) : %s", classname.c_str(), message.c_str());
```

Comment:
indentation

Reply:
hummm. That's odd. I thought I had that right. Will fix.